cobrautil.go

```diff
@@ -179,25 +198,30 @@ func OpenTelemetryRunE(flagPrefix string, prerunLevel zerolog.Level) CobraRunFun
 	}
 }
 
-func initJaegerTracer(endpoint, serviceName string) error {
-	exp, err := jaeger.New(jaeger.WithCollectorEndpoint(jaeger.WithEndpoint(endpoint)))
-	if err != nil {
-		return err
-	}
+func initOtlpTracer(exporter trace.SpanExporter, serviceName string) error {
```

Comment:
As this function is used by the Jaeger and OTLP Exporters, I would suggest to rename it to :
```suggestion
func initOtelTracer(exporter trace.SpanExporter, serviceName string) error {
```